package.json

```diff
@@ -34,7 +34,7 @@
   "homepage": "https://github.com/bigcommerce/paper-handlebars",
   "dependencies": {
     "@bigcommerce/handlebars-v4": "4.7.8",
-    "chrono-node": "^2.6.5",
+    "chrono-node": "2.8.0",
```

Comment:
FYI.  currently 2.8.1 is broken and temporary downgraded to 2.8.0

Reply:
i guess im also not following, why do we need the bump at all? your changes dont look to require this update at all

(and even in the entire repo, i can't find usages of this package)

Reply:
in terms of experiment I only mean that we simply can turn it off, if something is broken and SFR would use original version of handlebars. 
 `package-lock` is under [gitignore](https://github.com/bigcommerce/paper-handlebars/blob/master/.gitignore#L3) and i don't see it in the project structure as well.

Reply:
@bc-alexsaiannyi feel free to merge master into your branch, this change is already [there](https://github.com/bigcommerce/paper-handlebars/pull/378)

Reply:
great, thanks!